### ISSUE TITLE
GPU Overload

### ISSUE THREAD

**Issue by niemogewejsc** [Sk1erLLC/Patcher#115]
when minimizing the game with this mod, the graphics card starts to overload and makes "strange sounds" - squeaks (maybe only I have, I can't check)

video:
https://streamable.com/dkf0uk

**Comment by DJtheRedstoner** [comment]
Are you using Patcher's unfocused FPS feature? If so, what value is it set to?

**Comment by niemogewejsc** [comment]
no, but i am capped at 20fps on my graphics card if that matters

**Comment by niemogewejsc** [comment]
bruh why this option doesn't work ... with this modification
![image](https://user-images.githubusercontent.com/93555429/192118024-00dbc76c-f9e6-486b-a780-001c0b8ec985.png)
why so many fps are generated despite GPU limitations

**Comment by Sychic** [comment]
I can't seem to reproduce this issue, does removing that setting fix the usage issue?

**Comment by Scherso** [comment]
If you haven’t already, I would recommend removing Patcher, but keeping Essential installed to see if the issue prevails.
I’ve had a similar experience with capacitor squeal, but I resolved it by disabling Essential’s “Windowed Fullscreen” feature.
